doc/observable.md

```diff
@@ -277,21 +277,21 @@ Core Observable concerns:
 
 ### Creating Observables
 
-`Observable.create` is an alias for the `Observable` constructor, and it takes one argument: the `subscribe` function.
+The `Observable` constructor takes one argument: the `subscribe` function.
 
-The following example creates an Observable to emit the string `'hi'` every second to an Observer.
+The following example creates an Observable to emit the string `'hi'` every second to an subscriber.
 
 ```ts
 import { Observable } from 'rxjs';
 
-const observable = Observable.create(function subscribe(observer) {
+const observable = new Observable(function subscribe(subscriber) {
   const id = setInterval(() => {
-    observer.next('hi')
+    subscriber.next('hi')
   }, 1000);
 });
 ```
 
-<span class="informal">Observables can be created with `create`, but usually we use the so-called [creation operators](./overview.html#creation-operators), like `of`, `from`, `interval`, etc.</span>
+<span class="informal">Observables can be created with `new Observable`, but usually we use the so-called [creation operators](./overview.html#creation-operators), like `of`, `from`, `interval`, etc.</span>
```

Comment:
might want to stay away from the use of 'but' simply because it negates anything that is said before it and there is importance in what is said at first

Reply:
What about taking out `subscribe` function declaration on line 287 and put this above as arrow function?

```ts
const observable = new Observable(function subscribe(subscriber) {
  const id = setInterval(() => {
    subscriber.next('hi')
  }, 1000);
 ...
});
```

```ts
const subscribe = (subscriber) => {
  const id = setInterval(() => subscriber.next('hi'), 1000);
 ...
} 
const observable = new Observable(subscribe);
```

Reply:
It would be more readable if you taking out every callback function in these changes and declare a const as expression of arrow function 

```ts
const subscribe = (subscriber) => ...
const observer = new Observable(subscribe);
```